math/src/field/fields/u384_prime_field.rs

```diff
@@ -0,0 +1,208 @@
+use std::fmt::Debug;
+use std::marker::PhantomData;
+
+use crate::field::element::FieldElement;
+use crate::field::traits::IsField;
+use crate::unsigned_integer::UnsignedInteger384 as U384;
+
+pub trait HasU384Constant: Debug + Clone + Eq + PartialEq {
+    const VALUE: U384;
+}
+
+/// Type representing prime fields over unsigned 64-bit integers.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct U384PrimeField<MODULO: HasU384Constant> {
+    phantom: PhantomData<MODULO>,
+}
+
+pub type U384FieldElement<ORDER> = FieldElement<U384PrimeField<ORDER>>;
+
+impl<MODULO: HasU384Constant> IsField for U384PrimeField<MODULO> {
```

Comment:
Sames as [this comment](https://github.com/lambdaclass/lambdaworks/pull/10/files#r1100391011) here.